docker-containers/microservice_go/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM microservice
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+		g++ \
+		gcc \
+		git \
+		libc6-dev \
+		make \
+	&& rm -rf /var/lib/apt/lists/*
+
+ENV GOLANG_VERSION 1.5.3
+ENV GOLANG_DOWNLOAD_URL https://golang.org/dl/go$GOLANG_VERSION.linux-amd64.tar.gz
+ENV GOLANG_DOWNLOAD_SHA256 43afe0c5017e502630b1aea4d44b8a7f059bf60d7f29dfd58db454d4e4e0ae53
+
+RUN curl -fsSL "$GOLANG_DOWNLOAD_URL" -o golang.tar.gz \
+	&& echo "$GOLANG_DOWNLOAD_SHA256  golang.tar.gz" | sha256sum -c - \
+	&& tar -C /usr/local -xzf golang.tar.gz \
+	&& rm golang.tar.gz
+
+ENV GOPATH /go
+RUN ln -s /usr/local/go/bin/go /usr/local/bin/
```

Comment:
We can extend the PATH, instead of creating all these links:
ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH